src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx

```diff
@@ -0,0 +1,514 @@
+---
+title: "Analyzing Service Mesh Performance"
+subtitle: ""
+thumbnail: ./smp-light-text_2.png
+category: Service Mesh Performance
+tags:
+ - Service Mesh Performance
+ - GetNighthawk
+ - Meshery
+published: true
+---
+
+import { Link } from "gatsby";
+import wasmLogo from "./smp-light-text_2.png";
+import { TopicsWrapper } from "../Topics.style.js";
+
+<TopicsWrapper>
+  <div className="intro">
+    <p>Learn more about Service Mesh Performance from this article 
+      <Link className="blog" to="https://www.nxtbook.com/nxtbooks/ieee/bridge_issue3_2021/index.php#/p/16">Analyzing Service Mesh Performance</Link> - Published in issue 3 of IEEE Bridge October 2021  </div>
+

```

Comment:
missing a closing `</p>` tag here

Reply:
@warunicorn19 Thanks for the review !. Fixed.